### ISSUE TITLE
Add AdaBoost classifier

### ISSUE THREAD

**Issue by pkiage** [pkiage/tool-credit-risk-modelling#3]
Why:
Compare AdaBoost vs XGBoost training speed & performance given exponential vs gradience loss, respectively.

References:
- [Adaboost vs xgboost which is best and why ?](https://www.kaggle.com/general/218408)
- [Boosting and AdaBoost for Machine Learning](https://machinelearningmastery.com/boosting-and-adaboost-for-machine-learning/)
- [scikit-learn Ensemble Models - AdaBoost](https://scikit-learn.org/stable/modules/ensemble.html#adaboost)
